### ISSUE TITLE
Customization Stage Blocked on v1.6.4 & v1.7.0 Upgrade

### ISSUE THREAD

**Issue by richardkeit** [awslabs/landing-zone-accelerator-on-aws#469]
**Describe the bug**
During the prepare stage, pipeline files on customization portion:
~~~

$ /codebuild/output/src1439/src/s3/00/source/packages/@aws-accelerator/accelerator/node_modules/.bin/ts-node --transpile-only cdk.ts --require-approval never diff --config-dir /codebuild/output/src1439/src/s3/01 --partition aws --app cdk.out --stage customizations
--
247 | 2024-06-05 06:57:46.671 \| error \| accounts-config \| Account name not found for undefined. Validate that the emails in the parameter ManagementAccountEmail of the AWSAccelerator-InstallerStack and account configs (accounts-config.yaml) match the correct account emails shown in AWS Organizations.
248 | 2024-06-05 06:57:46.695 \| error \| toolkit \| configuration validation failed.
249 | 2024-06-05 06:57:46.702 \| error \| accelerator \| Runtime Error
250 | /codebuild/output/src1439/src/s3/00/source/packages/@aws-accelerator/accelerator/lib/accelerator.js:63
251 | throw new Error('Synthesis failed');
252 | ^
253 |  
254 | Error: Synthesis failed
255 | at process.<anonymous> (/codebuild/output/src1439/src/s3/00/source/packages/@aws-accelerator/accelerator/lib/accelerator.ts:93:9)
256 | at process.emit (node:events:529:35)
257 | at process.emit (node:domain:489:12)
258 | at process.emit.sharedData.processEmitHook.installedValue [as emit] (/codebuild/output/src1439/src/s3/00/source/node_modules/@cspotcode/source-map-support/source-map-support.js:745:40)
259 | at process._fatalException (node:internal/process/execution:158:25)


~~~

**To Reproduce**
Upgrade from v1.6.2 -> v1.6.4 / v.1.7.0
`customizations-config.yaml`:
~~~
homeRegion: &HOME_REGION ap-southeast-2 # replace ap-southeast-2 with actual home region
customizations:
  cloudFormationStacks:
    - deploymentTargets:
        accounts:
          - Management
      description: Creates the AWS Config recorder and enrols to the aggregator account.
      name: AWSAccelerator-Customization-AWSConfig
      regions:
        - *HOME_REGION
      runOrder: 1
      template: customizations/cfn-templates/aws-config-management-account-stack.yaml
      # Parameters for the AWS Config stack needs to be manually updated
      parameters:
        - name: AllConfigTopicName
          value: "aws-controltower-AllConfigNotifications"
        - name: AllSupported
          value: "true"
        - name: AuditBucketName
          value: "aws-controltower-logs-{{account LogArchive}}-ap-southeast-2"
        - name: Frequency
          value: "24hours"
        - name: HomeRegionName
          value: *HOME_REGION
        - name: IncludeGlobalResourceTypes
          value: "true"
        - name: IsHomeRegionInitialControlTowerRegion
          value: "true"
        - name: KMSKeyArn
          # The key alias that you want to use is <control-tower/cmk> if you are using the accelerator as is.
          # You can also confirm the policy document for the key, it should allow Config to use the key for generating key data and decryption.
          value: "arn:aws:kms:ap-southeast-2:{{account Management}}:alias/control-tower/cmk"
        - name: ManagedResourcePrefix
          value: "aws-controltower"
        - name: ResourceTypes
          value: ""
        - name: SecurityAccountId
          value: "{{account Audit}}"
      terminationProtection: true
~~~
_Template file is from `AWSControlTowerBP-BASELINE-CONFIG` stackset_

`accounts-config.yaml` (included for completeness):
~~~
mandatoryAccounts:
  - name: Management
    description: >-
      The management (primary) account. Do not change the name field for this
      mandatory account. Note, the account name key does not need to match the
      AWS account name.
    email: <prefix>+<email_tag>@<domain>.com.au
    organizationalUnit: Root
    warm: false
~~~

**Expected behavior**

**Please complete the following information about the solution:**

- [x] Version: v1.6.4

> To get the version of the solution, you can look at the description of the created AWS CloudFormation stack used to install the LZA (AWSAccelerator-InstallerStack). For example, "__(SO0199) Landing Zone Accelerator on AWS. **Version 1.5.1.**__". If the description does not contain the version information, you can look at the Parameters of the stack for the **RepositoryBranchName** as that should contain the version number.

- [x] Region: ap-southeast-2
- [ ] Was the solution modified from the version published on this repository?
- [ ] If the answer to the previous question was yes, are the changes available on GitHub?
- [x] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the services this solution uses?
- [ ] Were there any errors in the CloudWatch Logs?

**Screenshots**
If applicable, add screenshots to help explain your problem (please **DO NOT include sensitive information**).

**Additional context**

**Comment by richardkeit** [comment]
Removal of the `customization-config.yaml` allows the stage to continue.

**Comment by erwaxler** [comment]
Hey @richardkeit , thanks for raising this issue to the team. I was able to replicate this behavior and have implemented a fix that will be included in our next release. I'll keep this issue open until that release is available. Thank you for your support of the Landing Zone Accelerator!

**Comment by erwaxler** [comment]
Hi @richardkeit , a fix for this issue has been included in the [v1.7.1](https://github.com/awslabs/landing-zone-accelerator-on-aws/releases/tag/v1.7.1) release so I'll go ahead and close this issue. Thank you again for raising this issue to the Landing Zone Accelerator team!
